src/main/java/com/pelensky/contactmanager/ContactList.java

```diff
@@ -0,0 +1,16 @@
+package com.pelensky.contactmanager;
+
+import java.util.ArrayList;
+
+class ContactList {
```

Comment:
Not saying this is wrong, and I'm guessing you may have added to this class in subsequent commits, but I'd say this class is unnecessary here. An ArrayList<Contact> would have sufficed.